packages/react-accordion/src/components/AccordionItem/useAccordionItem.ts

```diff
@@ -18,9 +19,9 @@ import { AccordionContext } from '../Accordion/useAccordionContext';
  */
 export const accordionItemShorthandProps = [];
 
-export const accordionItemDescendantContext = createDescendantContext<AccordionItemDescendant>(
-  'AccordionItemDescendantContext',
-);
+export const accordionItemDescendantContext: React.Context<DescendantContextValue<
```

Comment:
If I will restore this change I am getting:

```
[11:59:13 AM] x Error: The expression contains an import() type, which is not yet supported by API Extractor:
office-ui-fabric-react/packages/react-accordion/lib/components/AccordionItem/useAccordionItem.d.ts:7:68
```

So we need this change only because of API extractor, correct?

Reply:
Unfortunately yes - quite a few of the changes are only because of API extractor.